src/constants/physics/CODATA2014.hpp

```diff
@@ -0,0 +1,89 @@
+#ifndef NJOY_CONSTANTS_PHYSICS_CODATA2014_HPP
+#define NJOY_CONSTANTS_PHYSICS_CODATA2014_HPP
+
+#include "constants/physics/CODATA.hpp"
+
+namespace njoy{
+namespace constants{
+namespace physics{
+
+struct PlanckLength : hana::type< PlanckLength >{
```

Comment:
This is not specific to CODATA

Reply:
I know, it was put there temporarily. I pushed up to ensure that it wouldn't get lost.